lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -303,7 +303,7 @@ def install_puppet_on(hosts, opts = options)
             # run_in_parallel option includes install
             run_in_parallel = run_in_parallel? opts, @options, 'install'
             block_on hosts, { :run_in_parallel => run_in_parallel } do |host|
-              if host['platform'] =~ /el-(5|6|7)/
+              if host['platform'] =~ /el-(5|6|7|8)/
```

Comment:
I wonder if it should assume it also works in the future (it'll likely work on EL9 too). Perhaps even ignore EL 4 and older. Nobody cares about those anymore.
```suggestion
              if host['platform'] =~ /el-(\d+)/
```

You could even merge it with fedora below if you also match on the first part.

Reply:
This makes sense to me, have pushed a change to merge with the fedora option as suggested